Tests/LibGfx/TestImageDecoder.cpp

```diff
@@ -143,3 +144,59 @@ TEST_CASE(test_ppm)
     auto frame = ppm.frame(0).release_value_but_fixme_should_propagate_errors();
     EXPECT(frame.duration == 0);
 }
+
+TEST_CASE(test_targa_bottom_left)
+{
+    auto file = Core::MappedFile::map("/res/html/misc/targasuite_files/oh-lena-bottom-left-uncompressed.tga"sv).release_value();
+    auto tga = Gfx::TGAImageDecoderPlugin((u8 const*)file->data(), file->size());
+    EXPECT(tga.frame_count());
```

Comment:
```suggestion
    EXPECT_EQ(tga.frame_count(), 1);
```

Since the TGA decoder always returns a frame count of 1.